.github/workflows/rockspec.template

```diff
@@ -0,0 +1,35 @@
+local repo_url = '$repo_url'
+
+rockspec_format = '3.0'
+package = '$package'
+version = modrev ..'-'.. specrev
+
+description = {
+  summary = '$summary',
+  labels = $labels,
+  homepage = '$homepage',
+  $license
+}
+
+dependencies = {
+	'lua >= 5.1',
+  'nui.nvim',
+  'nvim-web-devicons',
+  'plenary.nvim',
+}
```

Comment:
I haven't added the optional dependencies, because not all of them have been packaged for luarocks yet.
Not sure if you want them to be installed automatically default?

Reply:
Just let it be like this, I have plan to replace all of that dependencies later.